phac-nml/irida-next
# [ENHC0010012B] Workflow executions: End to end integration test

## What does this PR do and why?
_Describe in detail what your merge request does and why._

Adds a new github actions test which downloads/runs sapporo and tests a full workflow execution life cycle.

## Screenshots or screen recordings
_Screenshots are required for UI changes, and strongly recommended for all other pull requests._

## How to set up and validate locally
_Numbered steps to set up and validate the change are strongly suggested._

The test can be run locally with an updated Sapporo setup.

1. Make sure you have the latest version of the `irida-next` branch from `phac-nml/sapporo-service` https://github.com/phac-nml/sapporo-service/tree/irida-next
2. Use the updated commands to start sapporo https://github.com/phac-nml/sapporo-service/blob/irida-next/README.irida-next.md
3. Then run just the integration test
```bash
bin/rails test test/integration/sapporo.rb
```

## PR acceptance checklist
This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.
